duckduckgo/iOS
# Fix Settings > More from ddg section

Task/Issue URL: https://app.asana.com/0/414709148257752/1205324333927528/f

**Description**:

Introduced as part of [PR: Network Protection Design review](https://app.asana.com/0/0/1205278070827958). The layout of the TableView with 2 hide-able cells was proving tricky. Rather than spending too much time supporting AppTP when there’s no timeline on its development being resumed, I’ve just removed the AppTP parts from Settings.

**Steps to test this PR**:
1. Build this branch (needs to be Debug or Alpha)
2. Go to Settings
3. **Observe the More from DuckDuckGo section has a nice rounded footer**
4. Go to Debug Menu
5. Enable Internal User mode
6. Close Settings
7. Go back into Settings
8. **Observe the More from DuckDuckGo section has NetP and a nice rounded footer**

**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


—
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
